analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Constraints/NumberConstraint.cs

```diff
@@ -93,7 +93,7 @@ public static NumberConstraint From(BigInteger? min, BigInteger? max)
         }
         else
         {
-            return Empty;
+            return null;
```

Comment:
The `Empty` should be deleted too, it's not used anymore. `Empty`, `Zero` and `One`) were memory-saver for the most obvious reused values.